### ISSUE TITLE
meaning of random train/val shifting selection along time?

### ISSUE THREAD

**Issue by 2533245542** [zhouhaoyi/Informer2020#276]
In the paper, it was mentioned `Note that we preserves 10% validation data for each dataset, so all the experiments are conducted over 5 random train/val shifting selection along time and the results are averaged over the 5 runs.`

What does `random train/val shifting selection` mean? If assuming every time step in the validation set is ahead of the training set and the split ratio of train/val is fixed, there seems to be no room for randomization. Or does it mean it is possible that the time steps in train and validation sets are interleaved?

**Comment by cookieminions** [comment]
You can try to use rolling cross validation in time series prediction.
